common/apm/src/server.rs

```diff
@@ -1,7 +1,12 @@
 use axum::Router;
+use std::net::SocketAddr;
```

Comment:
See this lint https://github.com/cryptape/axon-internal/wiki/Axon-Develop-Lint#order-of-import